examples/spider-full/README.md

```diff
@@ -0,0 +1,10 @@
+## Usage
+
+```
+npx @empiricalrun/cli run
```

Comment:
merge with spider example. known gaps
- iteration speed will get slower with the full dataset: do we want to fix that? (e.g. add a way to sample the test dataset through rc.json)